assets/sass/modules/_header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+  padding-top: 60px;
+}
+.header__title {
+
+  & h1 {
+    padding  : 0;
+    margin   : 0;
+    @include fontSize(30px);
+  }
+}
+.site-description {
+    @include fontSize(16px);
+    color:#c0c0c0;
+    margin-bottom: 90px;
+}
+
+nav.header__nav {
+  width: 100%;
+  background: #253645;
+}
+ul.menu {
```

Comment:
This is how it looks now. It looks so ugly. Is it okay this way? Do you suggest any workaround for this? @mitogh